spec

```diff
@@ -5135,8 +5135,11 @@ beginseg
     name "ovl_En_Bomjima"
     compress
     include "build/src/overlays/actors/ovl_En_Bomjima/z_en_bomjima.o"
-    include "build/data/ovl_En_Bomjima/ovl_En_Bomjima.data.o"
+#ifdef NON_EQUIVALENT
+    include "build/src/overlays/actors/ovl_En_Bomjima/ovl_En_Bomjima_reloc.o"
+#else
     include "build/data/ovl_En_Bomjima/ovl_En_Bomjima.reloc.o"
+#endif
```

Comment:
```suggestion
    include "build/data/ovl_En_Bomjima/ovl_En_Bomjima.reloc.o"
```
If it's non-equivalent, then it'll break reloc afaik. So I think it's better to take it out completely. Others could confirm though?